src/index.html

```diff
@@ -8,6 +8,140 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container">
+      <div class="stars stars--0">
+        <div class="stars-star">
+
+        </div>
+
+        <div class="stars-star">
+
+        </div>
+
+        <div class="stars-star">
+
+        </div>
+
+        <div class="stars-star">
+
+        </div>
+
+        <div class="stars-star">
+
+        </div>
+      </div>
```

Comment:
```suggestion
      <div class="stars stars--0">
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
      </div>
```
Let's fix the formatting to improve readability. 
Also, pay attention to the naming
![image](https://github.com/mate-academy/layout_stars/assets/96409563/f22a972a-a0df-46c3-8aba-4dd9fbe213a5)